config/src/config/rpc_config.rs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::utils;
+use serde::{Deserialize, Serialize};
+use std::net::SocketAddr;
+
+#[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
+#[serde(default, deny_unknown_fields)]
+pub struct RpcConfig {
+    pub address: SocketAddr,
```

Comment:
Let's add a boolean on whether or not this is enabled. That way we can implicitly disable it by default in our validator and explicitly enable it only where we want to ... the full node?

Reply:
@davidiw: good idea! I was thinking about that and that's definitely something that we want to do long term. However, as I mentioned, first milestone is to deprecate and replace AC and AC is always enabled for now. I don't want to introduce divergence in behavior in this PR. AC on validators may still be temporary used in cluster-test